### ISSUE TITLE
fix generation of transparent background in PDF for 0.13.x

### ISSUE THREAD

**Issue by ashkulz** [wkhtmltopdf/wkhtmltopdf#2152]
This is a continuation of #2084 as Qt5 is broken due to [QTBUG-29040](https://bugreports.qt.io/browse/QTBUG-29040). Further investigation is required to fix and upstream this.

**Comment by ashkulz** [comment]
It looks like [additional code](http://stackoverflow.com/a/31188959/2536643) may be required to achieve this, which needs to be checked and committed for #2416 as well.

**Comment by bonny** [comment]
Have anyone managed to add the code (see below) that ashkultz mentions to wkhtml and build themself and that way managed to get transparency working? I really need transparent background for a project, but don't know where to add the code mentioned.

```
setAttribute(Qt::WA_TranslucentBackground);
setStyleSheet("background:transparent");
```

**Comment by ashkulz** [comment]
@bonny: it is already fixed in `0.12.3-dev`, this issue is for the `0.13` series.

**Comment by bonny** [comment]
@ashkulz: yes I know, but I need it working in the 0.13 series because I can't use the 0.12-series due to the font issues (https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2022). Sorry if I'm confusing things here..

**Comment by ashkulz** [comment]
You'll need to make the changes in `PdfConverterPrivate::beginPrintObject` (`src/lib/pdfconverter.cc`), do let me know if recompiling with the above changes does help.

**Comment by jeffreypoen** [comment]
@ashkulz: We also experience the transparancy problem, body is transparent, but the header and footer are not (white background).

Using the x64 Windows version, Windows Server 2012.

CMD: wkhtmltopdf --disable-smart-shrinking --margin-left 0.7cm --no-background --margin-right 0.7cm --margin-top 11.5cm --header-spacing 5 --header-html header.html --margin-bottom 5cm --footer-html footer.html body.html output.pdf

Then we add an full-color background PDF with pdftk

Recently we updated to 0.12.3, but the problem is stil there.
Could you give me a hint?

**Comment by ktrzos** [comment]
Bumping issue. Still no proper result. Even after upgrading wkhtmltopdf to v0.12.4.
